src/main/java/team18/team18_be/apply/dto/response/MandatoryResponse.java

```diff
@@ -1,6 +1,6 @@
 package team18.team18_be.apply.dto.response;
 
 public record MandatoryResponse(boolean resumeExistence, boolean visaExistence,
-                                boolean foreignerIdNumberExistence) {
+                                boolean foreignerIdNumberExistence,boolean signExistence) {
```

Comment:
컨벤션에 조금 더 신경써주세요.
IDE 설정을 통해 자동으로 컨벤션을 맞추는 것도 좋은 방법입니다.
```suggestion
                                boolean foreignerIdNumberExistence, boolean signExistence) {
```